os/arch/arm/src/amebad/chip.h

```diff
@@ -0,0 +1,142 @@
+/****************************************************************************
```

Comment:
Could you let me know why this is necessry?
Can we remove including chip.h instead of adding this?

Reply:
Yes, that would be better.
I will remove it